studio/components/StringInputWithCharacterCount.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Stack, Text } from "@sanity/ui";
```

Comment:
before we merge can you make sure it passes wcag 2.1? (plenty of studio is just bleh, but we can give it a try since it's a custom component
https://designsystem.digital.gov/components/character-count/accessibility-tests/

Reply:
Updated now to show the remaining characters, with a red text when over limit. Also added `aria-describedby` to announce allowed number of characters. `aria-live` has been set on the counter, but I have struggled to get the screen reader to announce it reliably (intercepted by text field description, which might be Sanity's fault).

Reply:
Yeah, I believe this is on Sanity. Great job though!